.github/workflows/build-notebooks-TEMPLATE.yaml

```
@@ -29,10 +29,20 @@ name: Build & Publish Notebook Servers (TEMPLATE)
+    {
+      "linux/amd64": "ubuntu-24.04",
+      "linux/arm64": "ubuntu-24.04-arm",
+      "linux/ppc64le": "ubuntu-24.04",
+      "linux/s390x": "ubuntu-24.04"
+    }
+
 jobs:
   build:
     # https://docs.github.com/en/actions/how-tos/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories
-    runs-on: ${{ inputs.platform == 'linux/arm64' && 'ubuntu-24.04-arm' || 'ubuntu-24.04' }}
+    runs-on: ${{ fromJSON(env.PLATFORM_RUNNERS)[inputs.platform] || 'no-such-runner' }}
```

actionlint (1.7.10)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)